packages/components/src/InputFile/InputFile.test.tsx

```diff
@@ -333,3 +333,173 @@ describe("PUT requests", () => {
     });
   });
 });
+
+describe("Content", () => {
+  function fetchUploadParams(file: File) {
+    return Promise.resolve({
+      key: file.name,
+      url: "https://httpbin.org/post",
+      fields: { secret: "🤫" },
+    });
+  }
+
+  it("renders default content in dropzone variation", () => {
+    render(
+      <InputFile
+        description="Description text"
+        hintText="Hint text"
+        getUploadParams={fetchUploadParams}
+      />,
+    );
+
+    const button = screen.getByRole("button", { name: "Upload file" });
+    expect(
+      screen.getByRole("button", { name: "Upload file" }),
+    ).toBeInTheDocument();
+    expect(button).not.toHaveClass("fullWidth");
+    expect(screen.queryByText("Description text")).toBeInTheDocument();
+    expect(screen.queryByText("Hint text")).toBeInTheDocument();
+  });
+
+  it("renders default content without hint text and description in small size", () => {
```

Comment:
In general I love that we are now testing the actual rendering of the component. I am surpised those tests didn't exist before